Data/go.mod

```diff
@@ -0,0 +1,3 @@
+module Data
```

Comment:
`mod` and `sum` files for a such small project should be stored in top directory (together with `main.go`).